currency/manager.go

```diff
@@ -497,7 +497,7 @@ func (p *PairsManager) SetDelimitersFromConfig() error {
 		}
 		for i, p := range []Pairs{s.Enabled, s.Available} {
 			for j := range p {
-				if p[j].Delimiter == cf.Delimiter {
+				if cf.Delimiter == "" || p[j].Delimiter == cf.Delimiter {
```

Comment:
Yeah I wasn't sure, I purged the entire function to see what actually goes on and its called in config in func `CheckExchangeConfigValues`. All pairs should contain a delimiter and the formatting will be overidden from the wrapper function in all cases. The config format and the request format are not configurable. So the only time it was needed was in a config upgrade but it would be flushed a delimiter set anyway. I will push a change up and see what breaks. Then you can call me a silly billy for doing such things.

Reply:
Yeah that broke everything to do with BTSE. 😆

Reply:
Edge case it seems when there is no config delimiter set. It doesn't do anything and fetch as normal and set as normal.
<img width="404" alt="image" src="https://github.com/user-attachments/assets/a301107b-516e-473c-9aa4-f9e67dc376fc">